erizo/src/erizo/rtp/SenderBandwidthEstimantionHandler.cpp

```diff
@@ -96,29 +99,54 @@ void SenderBandwidthEstimationHandler::read(Context *ctx, std::shared_ptr<dataPa
                 [last_sr](const std::shared_ptr<SrDelayData> sr_info) {
                 return sr_info->sr_ntp == last_sr;
                 });
-            // TODO(pedro) Implement alternative when there are no REMBs
-            if (received_remb_ && value != sr_delay_data_.end()) {
+            // TODO(pedro) Maybe implement better alternative when there are no REMBs
+            if (value != sr_delay_data_.end()) {
                 uint32_t delay = now_ms - (*value)->sr_send_time - delay_since_last_ms;
                 ELOG_DEBUG("%s message: Updating Estimate with RR, fraction_lost: %u, "
                     "delay: %u, period_packets_sent_: %u",
                     connection_->toLog(), chead->getFractionLost(), delay, period_packets_sent_);
                 sender_bwe_->UpdateReceiverBlock(chead->getFractionLost(),
                     delay, period_packets_sent_, now_ms);
                 period_packets_sent_ = 0;
-                updateEstimate();
+
+                uint64_t max_BW = 0;
+                for (auto &spatial_layer_node : stats_->getNode()["qualityLayers"].getMap()) {
+                  for (auto &temporal_layer_node :
+                      stats_->getNode()["qualityLayers"][spatial_layer_node.first.c_str()].getMap()) {
```

Comment:
IMO we're hacking the congestion control here... we're also capping the max bitrate value and it will go from the max layer bitrate to the remb when we receive it... I'm not sure if that's intended in webrtc's code.